sycl/test-e2e/Basic/max_work_group_query.cpp

```diff
@@ -0,0 +1,48 @@
+// RUN: %clangxx -fsycl -fsycl-targets=%sycl_triple  %s -o %t.out
+// REQUIRES: cuda || hip || level_zero
```

Comment:
Isn't `SYCL_EXT_ONEAPI_MAX_WORK_GROUP_QUERY` supported on OpenCL too? If so, please, just remove this line.

Reply:
It was mentioned in the doc that the host and OpenCL backend is not application to this [extension](https://github.com/intel/llvm/blob/cb91c232c661829b327c7e6e8232eb1d79100a98/sycl/doc/extensions/experimental/sycl_ext_oneapi_max_work_group_query.md?plain=1#LL91C3-L91C249):
``
If the device is the host or has an OpenCL back-end, the values returned - as they are not applicable - are the maximum values accepted at kernel submission (see `sycl/include/sycl/handler.hpp`) which are currently `std::numeric_limits<int>::max`
``

Reply:
That quote says that OpenCL returns bogus values, but they will pass the only check this test does. :)
```c++
  assert((group3D[0] <= group_max && group3D[1] <= group_max &&
          group3D[2] <= group_max) &&
         "Max work-group size of each dimension must be smaller than "
         "global work-group size");
```

I'm okay to skip testing this extension on OpenCL backend.